app/src/androidTest/java/com/rootstrap/android/tests/SignUpActivityTest.kt

```diff
@@ -20,13 +20,13 @@ import org.junit.Test
 @HiltAndroidTest
 class SignUpActivityTest : BaseTests() {
 
-    private lateinit var activity: SignUpActivity
-    private lateinit var scenario: ActivityScenario<SignUpActivity>
```

Comment:
Is this test class needed? I see that the the previous class has the same Activity and Scenario as the previous one, is it still needed? If so we can change the class name to one more descriptive. What do you think?

Reply:
I updated the test, check again, one for the SignUp and the other one for the SingIn tow different scenarios.